llcmapping.py

```diff
@@ -42,26 +41,23 @@ def __call__(self, da, ax=None, projection='robin', lon_0=-60, **plt_kwargs):
         vmax = plt_kwargs.pop('vmax', field.max())
         vmin = plt_kwargs.pop('vmin', field.min())
 
-        m = Basemap(projection=projection, lon_0=lon_0, resolution='c')
-        x,y = m(self.new_grid_lon, self.new_grid_lat) 
-        
+        m = plt.axes(projection=projection)
+        x,y = self.new_grid_lon, self.new_grid_lat
+
         # Find index where data is splitted for mapping
         split_lon_idx = round(x.shape[1]/(360/(lon_0 if lon_0>0 else lon_0+360)))
 
-        m.fillcontinents(color='lightgrey',lake_color='lightgray')
-        m.drawcoastlines(linewidth=1)
-        m.drawmeridians(np.arange(0,360,30))
-        m.drawparallels(np.arange(-90,90,30))
 
         p = m.pcolormesh(x[:,:split_lon_idx], y[:,:split_lon_idx], field[:,:split_lon_idx],
-                         vmax=vmax, vmin=vmin, **plt_kwargs)
+                         vmax=vmax, vmin=vmin, transform=cart.crs.PlateCarree(), zorder=1, **plt_kwargs)
         p = m.pcolormesh(x[:,split_lon_idx:], y[:,split_lon_idx:], field[:,split_lon_idx:],
-                         vmax=vmax, vmin=vmin, **plt_kwargs)
+                         vmax=vmax, vmin=vmin, transform=cart.crs.PlateCarree(), zorder=2, **plt_kwargs)
```

Comment:
Regarding my comment above, note that transform does *not* have to match projection of the axis.